tests/algorithms/test_particle_swarm_optimization.py

```diff
@@ -77,14 +77,11 @@ def test_pso_with_gambler(self):
         axl.seed(0)
         opt_vector, opt_objective_value = pso.swarm()
 
-        self.assertTrue(np.allclose(opt_vector, np.array([0.5488135, 0.71518937,
-                                                          0.60276338,
-                                                          0.54488318,
-                                                          0.4236548,
-                                                          0.64589411,
-                                                          0.43758721,
-                                                          0.891773])))
-        self.assertEqual(abs(opt_objective_value), 3)
+        self.assertTrue(np.allclose(opt_vector, np.array([[0.72231352, 0.29956341,
```

Comment:
What's happening here @Nikoleta-v3 ?

Reply:
This is actually on me from my previous `pr`. My method `player` was not implemented correctly because I was not passing the parameters (`op_plays` etc ) just the pattern/vector. 

But I just realised that I am still not getting it quite right. Let me fix this and add specific tests to reassure that the `player` method works fine for both algorithms.